src/middlewared/middlewared/plugins/tunables.py

```diff
@@ -24,6 +25,18 @@ class Config:
         datastore_prefix = 'tun_'
         datastore_extend = 'tunable.upper'
 
+    def __init__(self, *args, **kwargs):
+        super(TunableService, self).__init__(*args, **kwargs)
+        self.__default_sysctl = {}
+
+    @private
+    async def get_default_value(self, oid):
+        return self.__default_sysctl[oid]
+
+    @private
+    async def set_default_value(self, oid, value):
```

Comment:
Dont think you should overwrite default values, second run would overwrite default values with custom values?

Reply:
It should not occur with current implementation, but still, agree, that we can add protection in the method, itself.